tests/h/models/user_test.py

```diff
@@ -123,6 +123,57 @@ def test_userid_as_class_property_invalid_userid(db_session):
     assert result == []
 
 
+def test_userid_in_query(db_session):
+    fred = models.User(authority='example.net',
+                       username='fredbloggs',
+                       email='fred@example.net')
+    alice = models.User(authority='foobar.com',
+                        username='alicewrites',
+                        email='alice@foobar.com')
+    db_session.add_all([fred, alice])
+    db_session.flush()
+
+    result = (db_session.query(models.User)
+              .filter(models.User.userid.in_(['acct:fredbloggs@example.net',
+                                              'acct:alicewrites@foobar.com',
+                                              'acct:missing@bla.org']))
+              .all())
+
+    assert len(result) == 2
```

Comment:
I wonder if you can write this a one liner like `set(result) == set([fred, alice])` 🤔

Reply:
Sure it's possible. I think the error messages are easier to interpret when writing the test like I did.

Reply:
OK 👍